packages/docusaurus-plugin-debug/src/theme/DebugContent/index.js

```diff
@@ -5,52 +5,38 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React, {useState} from 'react';
+import React from 'react';
 
 import DebugLayout from '../DebugLayout';
 import DebugJsonView from '../DebugJsonView';
```

Comment:
Use `@theme/DebugJsonView` it will make it swizzlable.

Reply:
I believe we are making this page completely theme independent hence why we have a separated component inside a separate plugin.